cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -225,6 +225,11 @@ function (google_cloud_cpp_enable_features)
             if (NOT ("storage" IN_LIST GOOGLE_CLOUD_CPP_ENABLE))
                 add_subdirectory(google/cloud/storage)
             endif ()
+        elseif ("${feature}" STREQUAL "http-transcoding")
+            option(
+                GOOGLE_CLOUD_CPP_ENABLE_HTTP_TRANSCODING
+                "Enable building gRPC/HTTP transcoding libraries and executing accompanying tests."
```

Comment:
"executing accompanying tests"? I would just say "and their tests".

Reply:
Done.